grafana/provisioning/dashboards/CHT/cht_admin_api_express.json

```diff
@@ -0,0 +1,2738 @@
+{
+    "annotations": {
```

Comment:
Would CHT admins care about this? I suspect this dashboard targets core devs more than CHT admins. It has some deep nerdy metrics. I'm unclear when people would want to see them, but I know I want to see this. I suspect this is mostly useful when looking at performance problems and doing core dev work (?)

It may be easier to maintain if/when [this changes](https://grafana.com/grafana/dashboards/12230-node-js-dashboard/). 

If you agree, I'll update the name to `cht_coredev_api_express` or alike?

Reply:
This makes total sense to me!  Super valuable information for development or really deep troubleshooting/debugging, but not useful for day-to-day Admin monitoring.  I agree with keeping this as a separate dashboard and going with a name change to not include `admin`.